config/.base_rubocop.yml

```
@@ -100,9 +100,7 @@ Style/DefWithParentheses:
 
 # This might not be a style cop because errors happen due to the order of tests with nested classes/modules.
 Style/ClassAndModuleChildren:
-  Exclude:
-    - "test/**/*.rb"
-    - "spec/**/*.rb"
+Enabled: false
```

Indentation is incorrect.
```suggestion
  Enabled: false
```